packages/@lwc/perf-benchmarks/package.json

```diff
@@ -28,7 +28,7 @@
         "@best/runner-remote": "^8.1.3",
```

Comment:
These files are part of https://github.com/salesforce/lwc/pull/3484. Once that PR is merged, this PR will be rebased on top of the new master branch.